gomatic/fake.py

```diff
@@ -12,12 +12,14 @@ def __init__(self, text):
         self.text = text
         self.headers = {'x-cruise-config-md5': '42'}
         self.status_code = 200
-
+    def json(self):
+        return json.loads(self.text)
 
 class FakeHostRestClient(object):
-    def __init__(self, config_string, thing_to_recreate_itself=None):
+    def __init__(self, config_string, thing_to_recreate_itself=None, version="16.3.0"):
```

Comment:
maybe a little comment on why you have that set as "16.3.0" ? how can we keep that in sync with the versions we support? maybe a variable?